modules/oc-mirror-v2-about.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+// * updating/updating_a_cluster/updating_disconnected_cluster/mirroring-image-repository.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="installation-oc-mirror-v2-about_{context}"]
+= About the oc-mirror plugin V2
+
+
+You can use the oc-mirror OpenShift CLI (`oc`) plugin to mirror all required {product-title} content and other images to your mirror registry by using a single tool. 
+
+The purpose of a network enclave is to limit internal access to a portion of a network. A major difference between a DMZ or demilitarized zone and a network enclave is a DMZ allows inbound and outbound traffic access, where firewall boundaries are traversed. In an enclave, firewall boundaries are not traversed.
+
+This specific feature addresses use cases where mirroring is needed for several enclaves (disconnected environments), that are secured behind at least one intermediate disconnected network.
+
+You can mirror content for several enclaves, and centralizing it in a single internal registry. You can mirror contents directly from the internal centralized registry to enclaves without having to restart the mirroring from internet for each enclave. You can keep the volume of data transfered from one network stage to the other to a strict minimum, avoiding to transfer a blob or an image more than one time from one stage to another.
+
+
+It provides the following features:
+
+* Provides a centralized method to mirror {product-title} releases, Operators, helm charts, and other images.
+* Maintains update paths for {product-title} and Operators.
+* Uses a declarative image set configuration file to include only the {product-title} releases, Operators, and images that your cluster needs.
+* Performs incremental mirroring, which reduces the size of future image sets.
+* Prunes images from the target mirror registry that were excluded from the image set configuration since the previous execution.
+* Optionally generates supporting artifacts for OpenShift Update Service (OSUS) usage.
+* Use the existing workflow use case/s (as in v1)
+- Mirror to Disk
+- Disk to Mirror
+- Mirror to Mirror
+* Improve overall user experience
+- Use of local caching mechanism
+- Continue where you left off
+- Save time for mirroring (especially for multiple enclaves)
+- Geared for gitops (fail fast)
```

Comment:
This is only a personal opinion: this are is part of v2 design goals. Same goes for "Continue where you left off" 
I'm not very certain we've reached them with 4.16. 
Personally, I would leave these to later versions where we have some tangible proof that we can achieve it

Reply:
Understood.